diff --git a/configure/CONFIG_SITE b/configure/CONFIG_SITE
index 40caaad..095fe20 100644
--- a/configure/CONFIG_SITE
+++ b/configure/CONFIG_SITE
@@ -1,29 +1,26 @@
 # CONFIG_SITE
 
 # Make any application-specific changes to the EPICS build
-# configuration variables in this file.
+#   configuration variables in this file.
 #
 # Host/target specific settings can be specified in files named
-#  CONFIG_SITE.$(EPICS_HOST_ARCH).Common
-#  CONFIG_SITE.Common.$(T_A)
-#  CONFIG_SITE.$(EPICS_HOST_ARCH).$(T_A)
+#   CONFIG_SITE.$(EPICS_HOST_ARCH).Common
+#   CONFIG_SITE.Common.$(T_A)
+#   CONFIG_SITE.$(EPICS_HOST_ARCH).$(T_A)
 
 # Set this when you only want to compile this application
-# for a subset of the cross-compiled target architectures
-# that Base is built for.
-#CROSS_COMPILER_TARGET_ARCHS = RTEMS-beatnik
-#CROSS_COMPILER_TARGET_ARCHS = RTEMS-uC5282
-#CROSS_COMPILER_TARGET_ARCHS = RTEMS-pc586
-#CROSS_COMPILER_TARGET_ARCHS = RTEMS-beatnik linuxRT_glibc-x86_64
+#   for a subset of the cross-compiled target architectures
+#   that Base is built for.
+#CROSS_COMPILER_TARGET_ARCHS = vxWorks-68040
+
+# To install files into a location other than $(TOP) define
+#   INSTALL_LOCATION here.
+#INSTALL_LOCATION=</path/name/to/install/top>
 
 # Set this when your IOC and the host use different paths
-# to access the application. This will be needed to boot
-# from a Microsoft FTP server or with some NFS mounts.
+#   to access the application. This will be needed to boot
+#   from a Microsoft FTP server or with some NFS mounts.
 # You must rebuild in the iocBoot directory for this to
-# take effect.
-#IOCS_APPL_TOP = <path to application top as seen by IOC>
-
-# If you don't want to install into $(TOP) then
-# define INSTALL_LOCATION here
-#INSTALL_LOCATION=<fullpathname>
+#   take effect.
+#IOCS_APPL_TOP = </IOC/path/to/application/top>
 
diff --git a/configure/RULES b/configure/RULES
index 1f2cb5d..6d56e14 100644
--- a/configure/RULES
+++ b/configure/RULES
@@ -1,6 +1,6 @@
 # RULES
 
-include $(EPICS_BASE)/configure/RULES
+include $(CONFIG)/RULES
 
 # Library should be rebuilt because LIBOBJS may have changed.
 $(LIBNAME): ../Makefile
diff --git a/configure/RULES_TOP b/configure/RULES_TOP
index 0620f08..9899b6c 100644
--- a/configure/RULES_TOP
+++ b/configure/RULES_TOP
@@ -1,3 +1,4 @@
 #RULES_TOP
-include $(EPICS_BASE)/configure/RULES_TOP
-
+include $(CONFIG)/RULES_TOP
+UNINSTALL_DIRS += $(INSTALL_RESTORE)
+UNINSTALL_DIRS += $(INSTALL_ARCHIVE)
